repos/system_upgrade/el7toel8/models/satellite.py

```diff
@@ -22,5 +22,7 @@ class SatelliteFacts(Model):
 
     has_foreman = fields.Boolean(default=False)
     """Whether or not foreman is installed on this system"""
+    has_katello_installer = fields.Boolean(default=True)
```

Comment:
Hmm are you sure that you want this by default to be True?

Reply:
To me it seems like rather something to be default False, but well then again you wouldn't send the message in the first place if you wouldn't have checked for it. 

This is just a question not a request for changes.

Reply:
I have no strong opinion about the default, as all further actions that would take this field into account are guarded by the fact that the `SatelliteFacts` message was sent at all *and* the `has_foreman` field is true. And if that happens, this field has been set explicitly by the `satellite_upgrade_facts` actor to either `True` or `False`.